modules/3d-tiles/src/tileset/tileset-3d.js

```diff
@@ -380,16 +380,17 @@ export default class Tileset3D {
     this.zoom = getZoom(root.boundingVolume);
   }
 
-  _initStats() {
+  _initializeStats() {
     this.stats.get(TILES_TOTAL);
```

Comment:
Nit: I still think we should support adding these in  a constructor option

```
new Stats({stats: {
  TILES_LOADING: 'count',
  ...
}});
```

Reply:
https://github.com/uber-web/probe.gl/pull/92/files